src/runtime_src/core/common/info_aie.cpp

```diff
@@ -70,14 +70,35 @@ addnodelist(const std::string& search_str, const std::string& node_str,
 static void
 populate_aie_dma(const boost::property_tree::ptree& pt, boost::property_tree::ptree& pt_dma)
 {
+  boost::property_tree::ptree fifo_pt;
   boost::property_tree::ptree mm2s_array;
   boost::property_tree::ptree s2mm_array;
   boost::property_tree::ptree empty_pt;
 
+  // Extract FIFO COUNT information
+  // Sample sysfs entry for DMA
+  // cat /sys/devices/platform/<aie_path>/aiepart_0_50/47_0/dma
+  // ichannel_status: mm2s: idle|idle, s2mm: idle|idle
+  // queue_size: mm2s: 0|0, s2mm: 0|0
+  // queue_status: mm2s: okay|okay, s2mm: okay|okay
+  // current_bd: mm2s: 0|0, s2mm: 0|0
+  // fifo_len: 0|0
+  int id = 0;
+  for (const auto& node : pt.get_child("dma.fifo_len", empty_pt)) {
+    std::string counter_id = "Counter";
+    boost::property_tree::ptree fifo_counter;
+    counter_id += std::to_string(id++);
+    fifo_counter.put("counter_id", counter_id);
```

Comment:
This is an array of items.  Why is there a need to create the "counter_id".  It shouldn't be needed for the "index" value is the ID of the counter.

Reply:
Fixed. Thanks